src/drivers/dshot/DShot.cpp

```diff
@@ -308,9 +97,9 @@ DShotOutput::set_mode(Mode mode)
 	 * listening and mixing; the mode just selects which of the channels
 	 * are presented on the output pins.
 	 */
-	switch (mode) {
+	switch (new_mode) {
```

Comment:
Please keep this in sync with https://github.com/PX4/PX4-Autopilot/blob/master/src/drivers/pwm_out/PWMOut.cpp#L103

Reply:
OK. Thanks!